README.md

```diff
@@ -243,6 +245,26 @@ Additionally, users can utilize the MiNiFi Toolkit Converter (version 0.0.1 - sc
                     Port: 10001
                     Host Name: localhost
 
+### Site2Site Security Configuration
+
+    in minifi.properties 
```

Comment:
We should stub these out as defaults in minifi.properties.  No values, but at least provide a quick way of having them be readily available.